mitmproxy/addons/export.py

```diff
@@ -106,6 +107,65 @@ def httpie_command(f: flow.Flow) -> str:
     return cmd
 
 
+def python_requests_command(f: flow.Flow) -> str:
+    # Copy as Python Requests
+    request = cleanup_request(f)
+    request = pop_headers(request)
+
+    def escape_quotes(s: str) -> str:
+        return s.replace('"', '\\"')
+
+    code = "import requests\n\n"
+
+    code += f'url = "{request.pretty_url}"\n'
+
+    # parse cookie
+    code += "cookies = {\n"
+    for k, v in request.cookies.items():
+        code += f'    "{k}": "{escape_quotes(v)}",\n'
+    code += "}\n"
+
+    # parse header
+    request.headers.pop("cookie", None)
+    code += "headers = {\n"
+    for k, v in request.headers.items():
+        code += f'    "{k}": "{escape_quotes(v)}",\n'
+    code += "}\n"
```

Comment:
Can we use pformat here instead of manually constructing stuff?

Reply:
Similar to the above, this will create
```
headers = Headers[(b'header', b'qvalue')]
```